pandas/core/reshape/melt.py

```diff
@@ -386,7 +386,7 @@ def wide_to_long(df, stubnames, i, j, sep="", suffix=r'\d+'):
     8      3      3     2.1     2.9
 
     >>> l = pd.wide_to_long(df, stubnames='ht', i=['famid', 'birth'], j='age',
-                            sep='_', suffix='\w')
+                            sep='_', suffix='\w+')
```

Comment:
Need `...` under the `>>>`.

Reply:
Errors:
```python
################################### Doctests ###################################
################################################################################

**********************************************************************
Line 94, in pandas.wide_to_long
Failed example:
    df
Expected:
       birth  famid  ht1  ht2
    0      1      1  2.8  3.4
    1      2      1  2.9  3.8
    2      3      1  2.2  2.9
    3      1      2  2.0  3.2
    4      2      2  1.8  2.8
    5      3      2  1.9  2.4
    6      1      3  2.2  3.3
    7      2      3  2.3  3.4
    8      3      3  2.1  2.9
Got:
       famid  birth  ht1  ht2
    0      1      1  2.8  3.4
    1      1      2  2.9  3.8
    2      1      3  2.2  2.9
    3      2      1  2.0  3.2
    4      2      2  1.8  2.8
    5      2      3  1.9  2.4
    6      3      1  2.2  3.3
    7      3      2  2.3  3.4
    8      3      3  2.1  2.9
**********************************************************************
Line 154, in pandas.wide_to_long
Failed example:
    df # doctest: +NORMALIZE_WHITESPACE, +ELLIPSIS
Expected:
       A(quarterly)-2010  A(quarterly)-2011  B(quarterly)-2010  ...
    0           0.548814           0.544883           0.437587  ...
    1           0.715189           0.423655           0.891773  ...
    2           0.602763           0.645894           0.963663  ...
       X  id
    0  0   0
    1  1   1
    2  1   2
Got:
       A(quarterly)-2010  A(quarterly)-2011  B(quarterly)-2010  B(quarterly)-2011  X  id
    0           0.548814           0.544883           0.437587           0.383442  0   0
    1           0.715189           0.423655           0.891773           0.791725  1   1
    2           0.602763           0.645894           0.963663           0.528895  1   2
**********************************************************************
Line 195, in pandas.wide_to_long
Failed example:
    df
Expected:
       birth  famid  ht_one  ht_two
    0      1      1     2.8     3.4
    1      2      1     2.9     3.8
    2      3      1     2.2     2.9
    3      1      2     2.0     3.2
    4      2      2     1.8     2.8
    5      3      2     1.9     2.4
    6      1      3     2.2     3.3
    7      2      3     2.3     3.4
    8      3      3     2.1     2.9
Got:
       famid  birth  ht_one  ht_two
    0      1      1     2.8     3.4
    1      1      2     2.9     3.8
    2      1      3     2.2     2.9
    3      2      1     2.0     3.2
    4      2      2     1.8     2.8
    5      2      3     1.9     2.4
    6      3      1     2.2     3.3
    7      3      2     2.3     3.4
    8      3      3     2.1     2.9
```

and after my attempted fix, 
```python3
################################################################################
################################### Doctests ###################################
################################################################################

**********************************************************************
Line 154, in pandas.wide_to_long
Failed example:
    df # doctest: +NORMALIZE_WHITESPACE, +ELLIPSIS
Expected:
       A(quarterly)-2010  A(quarterly)-2011  B(quarterly)-2010  ...
    0           0.548814           0.544883           0.437587  ...
    1           0.715189           0.423655           0.891773  ...
    2           0.602763           0.645894           0.963663  ...
       X  id
    0  0   0
    1  1   1
    2  1   2
Got:
       A(quarterly)-2010  A(quarterly)-2011  B(quarterly)-2010  B(quarterly)-2011  X  id
    0           0.548814           0.544883           0.437587           0.383442  0   0
    1           0.715189           0.423655           0.891773           0.791725  1   1
    2           0.602763           0.645894           0.963663           0.528895  1   2
```
The ELLIPSIS is not being considered or ignored by doctest as it ought to be. Perhaps another PR?

Reply:
The doctest, and hence the entire pipeline is failing only due to this ELLIPSIS error @TomAugspurger any suggestions on what to do? (If I put all columns, `pylint` and `flake8` fails because it exceeds 79 chars)

Reply:
Answered in 
https://github.com/pandas-dev/pandas/pull/25736/files#r265931954